frameworks/Python/.gitignore

```diff
@@ -1,2 +1,6 @@
 uwsgi_params
 *.pid
+
+# ignore virtual environments
+pyvenv/
+venv/
```

Comment:
I want all text file ends with newline.
(Although this is not requirement for this project.)